packages/http/src/validator/__tests__/__snapshots__/functional.spec.ts.snap

```diff
@@ -18,6 +18,7 @@ Object {
       "message": "should match format \\"date-time\\"",
       "path": Array [
         "header",
+        "['x-todos-publish']",
```

Comment:
This change to the snapshot looks like making sense as well, doesn't it?

Reply:
I'm not too sure why it's an array and not a string.

Reply:
Ah ok, it's AJV that is doing so. We'll not investigate on this.